content/utmt/rooms/creating new rooms/preperation/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Preperation
+weight: 0
+resources: 
+  - name: roomSetup
+    src: "roomSetup.png"
+    title: This is approximately how the room should look after the preparation
+  - name: roomOnCreate
+    src: "roomOnCreate.png"
+    title: This is how it will look when first adding a new room
+  - name: viewsSetup
+    src: "viewsSetup.png"
+    title: This is how the Views should look like.
+  - name: screenGuideSetup
+    src: "screenGuideSetup.png"
+    title: This is what it should look like after setting up the `bgScreenGuide` element 
+---
+
+Before we set up tiles or textures, we must first make some preperations. To get started, create a new room by right-clicking on `Rooms` in the viewer on the left and then click `add`. Now you will see 3 things;
+- A list of all objects that are in your room.
+- A set of values, corresponding to the element that you currently have selected.
+- A visual overview of your room. If you're adding a new room, this will be completely black with a grey grid.
```

Comment:
```suggestion
- At the top left, a list of all objects that are in your room.
- At the top right, a set of values, corresponding to the element that you currently have selected.
- At the bottom, a visual overview of your room. If you're adding a new room, this will be completely gray.
```